app/repositories/application/GeneralApplicationRepoBSONToModelHelper.scala

```diff
@@ -107,23 +107,11 @@ trait GeneralApplicationRepoBSONToModelHelper {
     val applicationId = doc.getAs[String]("applicationId").getOrElse("")
     val progress: ProgressResponse = findProgress(doc, applicationId)
 
-    CandidateProgressReport(applicationId, Some(ProgressStatusesReportLabels.progressStatusNameInReports(progress)), schemes.getOrElse(List.empty[SchemeType]), disability, onlineAdjustments,
+    CandidateProgressReport(applicationId, Some(ProgressStatusesReportLabels.progressStatusNameInReports(progress)),
+      schemes.getOrElse(List.empty[SchemeType]), disability, onlineAdjustments,
```

Comment:
schemes.getOrElse(List.empty[SchemeType]) could be replaced with Nil here as the type can be inferred. I appreciate that you just reformatted the code, but in future we can use that for readability